File: GameTemetry/DTOs/PerformanceMetricDto.cs
namespace GameTemetry.DTOs
{
    public class PerformanceMetricDto
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int ExerciseId { get; set; }
        public string ExerciseName { get; set; } = null!;
        public decimal TotalVolume { get; set; }
        public decimal AverageReps { get; set; }
        public decimal AverageRIR { get; set; }
        public decimal MaxWeight { get; set; }
        public int WorkoutsCount { get; set; }
        public DateTime LastPerformed { get; set; }
    }
}

File: GameTemetry/GameTelemetry-App/src/app/core/models/performance-metric.dto.ts
export interface PerformanceMetricDto {
  id: number;
  userId: number;
  exerciseId: number;
  exerciseName: string;
  totalVolume: number;
  averageReps: number;
  averageRIR: number;
  maxWeight: number;
  workoutsCount: number;
  lastPerformed: string;
}
